# goal definition
# name of the behavior
string interacting_action
string optional_data

---
# result definition
string interacting_action
bool interaction_continue
string message

---
# feedback
string interacting_action
string interaction_state